packages/core/content-type-builder/admin/src/components/FormModal/contentType/form.js

```diff
@@ -12,25 +12,42 @@ const nameField = {
 const forms = {
   advanced: {
     default() {
+      const items = [
+        {
+          intlLabel: {
+            id: getTrad('contentType.draftAndPublish.label'),
+            defaultMessage: 'Draft & publish',
+          },
+          description: {
+            id: getTrad('contentType.draftAndPublish.description'),
+            defaultMessage: 'Allows writing a draft version of an entry, before it is published',
+          },
+          name: 'draftAndPublish',
+          type: 'toggle-draft-publish',
+          validations: {},
+        },
+      ];
+
+      if (window.strapi.features.isEnabled(window.strapi.features.REVIEW_WORKFLOWS)) {
+        items.push({
+          intlLabel: {
+            id: getTrad('contentType.reviewWorkflows.label'),
+            defaultMessage: 'Review Workflow',
```

Comment:
@maevalienard We are now in the CTB: I saw in figma you are using the plural here, which seems to be inconsistent to me. Do you agree we should keep it in singular?

Reply:
It seems that we have agreed to call the feature "Review workflowS" so I would stay consistent to this agreement rather than the rest of the interface :)

Reply:
Oh I see 🤔 This wasn't part of the user stories - they were all still referring to singular. Thanks!